tedana/tests/test_decay.py

```diff
@@ -75,14 +73,10 @@ def test_smoke_fit_decay():
     mask = np.random.randint(2, size=n_samples)  # generate binary mask of random 0s and 1s
     masksum = np.random.random((n_samples))
     fittype = 'loglin'
-    t2s_limited, s0_limited, t2ss, s0vs, t2s_full, s0_full = me.fit_decay(
-                                                            data, tes,
-                                                            mask, masksum,
-                                                            fittype)
+    t2s_limited, s0_limited, t2s_full, s0_full = me.fit_decay(
+        data, tes, mask, masksum, fittype)
```

Comment:
Same thing here - perhaps I'm missing something, but shouldn't masksum here be adaptive mask, just for consistency?